beacon_chain/utils/hashing/src/merkle_root.rs

```diff
@@ -0,0 +1,31 @@
+use types::{Hash256}
+use hashing::canonical_hash;
+use std::iter::range_step;
+
+fn merkle_root(values: Vec<T>) -> Hash256 {
+    let mut o = vec![0; values.len()];
+    o.append(values);
```

Comment:
values may need to be converted into a list of their hashes here
 if they are not already ssz or hash values (we need to check this)
 so this might be something like:
         
	 for v in values {
	    value_hashes.push(canonical_hash(v);
	 }
	 o.append(value_hashes);